content/en/storage-providers/advanced-configurations/workers.md

```diff
@@ -0,0 +1,685 @@
+lead: "Lotus has a lot of advanced configurations you can tune to optimize your storage provider setup. This guide explains the advanced configuration options for Seal and PoSt workers"
+draft: false
+menu:
+    storage-providers:
+        parent: "storage-providers-advanced-configurations"
+weight: 110
+toc: true
+---
+
+While the Lotus workers have very reasonable default settings, some storage providers might want to fine tune some advanced configurations according to their setup.
+
+## Resource allocation in Seal Workers
+
+Each **Seal Worker** can potentially run multiple tasks in available slots. Each slot is called a _window_. The number of available windows per worker is determined by the requirements of the sealing tasks being allocated to the worker and its available system resources, including:
+
+- Number of CPU threads the task will use.
+- Minimum amount of RAM required for good performance.
+- Maximum amount of RAM required to run the task, where the system can swap out part of the RAM to disk, and the performance won't be affected too much.
+- Whether the system has a GPU.
+
+### Task resource table
+
+The default resource table lives in [resources.go](https://github.com/filecoin-project/lotus/blob/master/extern/sector-storage/storiface/resources.go) and can be edited to tune the scheduled behavior to fit specific sealing clusters better.
+
+Default resource value table. Some of these values are _fairly_ conservative:
+
+| Sector size | Task Type  | Threads | Min RAM | Min disk space| GPU        |
+|-------------|------------|---------|---------|------------|------------|
+|     32G     | AddPiece   | 1*      | 4G      | 4G         |            |
+|             | PreCommit1 | 1**     | 56G     | 64G        |            |
+|             | PreCommit2 | 92%***  | 15G     | 15G        | If Present |
+|             | Commit1    | 0****   | 1G      | 1G         |            |
+|             | Commit2    | 92%***  | 32G+30G | 32G+150G   | If Present |
+|     64G     | AddPiece   | 1*      | 8G      | 8G         |            |
+|             | PreCommit1 | 1**     | 112G    | 128G       |            |
+|             | PreCommit2 | 92%***  | 30G     | 30G        | If Present |
+|             | Commit1    | 0****   | 1G      | 1G         |            |
+|             | Commit2    | 92%***  | 64G+60G | 64G+190G   | If Present |
+
+\* AddPiece can use multiple threads, it's likely that this value will change in near future\
+** When used with the `FIL_PROOFS_USE_MULTICORE_SDR=1` env var, PreCommit1 can use multiple cores (up to the number of cores sharing L3 caches)\
+*** Depending on the number of available threads, this value means:
+
+```
+ 12  * 0.92 = 11
+ 16  * 0.92 = 14
+ 24  * 0.92 = 22
+ 32  * 0.92 = 29
+ 64  * 0.92 = 58
+ 128 * 0.92 = 117
+```
+
+**** The Commit1 step is very cheap in terms of CPU time and blocks the Commit2 step. Allocating it to zero threads makes it more likely it will be scheduled with higher priority.
```

Comment:
```suggestion
\**** The Commit1 step is very cheap in terms of CPU time and blocks the Commit2 step. Allocating it to zero threads makes it more likely it will be scheduled with higher priority.
```